pressbooks/pressbooks
# New dynamic thema subjects

Related Issue #2105 

This PR depends on `https://github.com/pressbooks/symbionts-thema` to download the new thema subjects for each supported language

Thema subjects json file will be downloaded on demand when the book language is updated if there is no thema file for the selected language an en fallback will be loaded instead.

1. Thema files were deleted from the plugin only english version is loaded by default to provide a safe fallback.

### How to test

1. Change the language of your book and review if the subject themas are being loaded.
2. Try to select a unsupported thema language to see if the default english are loaded.
